README.md

```diff
@@ -7,12 +7,14 @@ a read-only access to the repository.
 
 ```hcl
 module "fluxcd" {
-  source                     = "github.com/lassizci/terraform-eks-fluxcd-sops?ref=v0.14"
+  source                     = "github.com/neondatabase/terraform-eks-fluxcd-sops?ref=v0.16"
   path                       = "./clusters/dev"
   controller_ssh_public_key  = file("./deploy-key.pub")
   controller_ssh_private_key = file("./deploy-key.priv")
   controller_ssh_known_hosts = "github.com ecdsa-sha2-nistp256 AAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlzdHAyNTYAAABBBEmKSENjQEezOmxkZMy7opKgwFB9nkt5YRrYMjNuG5N87uRgg6CLrbo5wAdT/y6v0mKV0U2w0WZ2YB/++Tpockg="
-  irsa_role_arn              = "arn:aws:iam::123456789012:role/fluxcd-irsa-role"
+  service_account_annotations = {
+    "eks.amazonaws.com/role-arn": "arn:aws:iam::123456789012:role/fluxcd-irsa-role"
```

Comment:
since this is a `map(string)`, should it be
```
   "eks.amazonaws.com/role-arn" = "arn:aws:iam::123456789012:role/fluxcd-irsa-role"
```

Reply:
Good catch, thank you! Force of habit from jsonnet 😄 Fixed!